client/wfnews-war/src/main/angular/src/app/components/draggable-panel/draggable-panel.component.ts

```diff
@@ -769,4 +775,21 @@ type = 'evac-order';
     const formattedDate: string = date.toLocaleDateString('en-US', options);
     return formattedDate;
   }
+
+  fitPolygonToMap(polygonData: number[][], viewerMap: L.Map): L.Polygon {
+    const swappedPolygonData: number[][] = polygonData.map(([latitude, longitude]) => [longitude, latitude]);
+    const polygonShape = L.polygon(
```

Comment:
Why are we adding the polygon data to the map? Does it get removed later? I didn't see a removal. If this is just a zoom to bounds, we don't need to add it